app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Recipes</h1>
+<% @recipe.each_with_index do |recipe,index| %>
+    <div class="recipes">
+        <%= link_to recipe_path(recipe.id) do %>
+            <ul class="recipes-title" >
+                <li>Recipe:<%= index + 1 %></li>
+                <li><%= recipe.name %></li>
+            </ul>
+            <ul class="recipes-desc">
+                <li><%= recipe.description%></li>
+            </ul>
+        <% end %>
+        <%= link_to 'Remove', recipe_path(recipe), data: {
+                                turbo_method: :delete,
+                                turbo_confirm: "Are you sure?"
+                            }  %>
+    </div>
```

Comment:
:red_circle: Please make sure your page follows the given wireframe. Your remove position should be positioned to the left side and the description to the right side to match the wireframe design

Wireframe
![recipe](https://github.com/huzaifakhan0308/Recipe-app/assets/26736582/f239f634-ef7c-4bf3-8433-6d855480b953)

Your Page

![right](https://github.com/huzaifakhan0308/Recipe-app/assets/26736582/12c73897-6594-49ae-9e54-dd8bc7364b1f)